wpilibj/publish.gradle

```diff
@@ -51,6 +51,7 @@ task sourcesJar(type: Jar, dependsOn: classes) {
 task javadoc(type: Javadoc, overwrite: true) {
     javadoc.options.links("http://docs.oracle.com/javase/8/docs/api/")
     options.addStringOption "tag", "pre:a:Pre-Condition"
+    source = sourceSets.main.allJava
 //    options.addStringOption('Xdoclint:accessibility,syntax,reference,html', '-quiet')
```

Comment:
This fails to build without uncommenting the Xdoclint options

Reply:
Hmm it had successfully built on my side, but fixed.